server/channels/app/imports/import_validators_test.go

```diff
@@ -1541,6 +1541,36 @@ func TestImportValidateEmojiImportData(t *testing.T) {
 	}
 }
 
+func TestImportValidateThreadFollowerImportData(t *testing.T) {
+	testCases := []struct {
+		testName       string
+		lastViewed     *int64
+		unreadMentions *int64
+		user           *string
+		expectError    bool
+	}{
+		{"success", model.NewPointer(int64(0)), model.NewPointer(int64(0)), model.NewPointer("user1"), false},
+		{"missing user", model.NewPointer(int64(0)), model.NewPointer(int64(0)), nil, true},
```

Comment:
Coverage-wise, we are missing the "pass nil ThreadFollowerImportData" case and the empty user string. It's not a huge deal, but it shouldn't be hard to add them.

Reply:
I added those, and I reformatted the test a bit to make that work